src/main/java/org/htmlunit/cyberneko/HTMLScanner.java

```diff
@@ -1893,6 +1897,14 @@ void restorePosition(final int originalOffset, final int originalColumnNumber, f
         int getCharacterOffset() {
             return characterOffset_;
         }
+
+        public void setIsProcessingPlaintext(boolean processingPlaintext) {
```

Comment:
how come the other methods are non-public, shouldn't this also be non-public by same convention?